site-manage/site-manage-tool/tool/src/java/org/sakaiproject/site/tool/SiteAction.java

```diff
@@ -15409,6 +15407,15 @@ private void redirectCourseCreation(ParameterParser params, SessionState state,
 				state.setAttribute(STATE_TEMPLATE_INDEX, "37");
 			}
 		}
+
+		if( "true".equalsIgnoreCase( ServerConfigurationService.getString( SAK_PROP_CONT_NO_ROSTER_ENABLED, "true" ) ) )
+		{
+			state.setAttribute( VM_CONT_NO_ROSTER_ENABLED, Boolean.TRUE );
+		}
+		else
+		{
+			state.setAttribute( VM_CONT_NO_ROSTER_ENABLED, Boolean.FALSE);
+		}
```

Comment:
Hello @teetoppz28 , thanks for looking into this, I keep what it was in the original patch by Brian Jones in [SAK-28990](https://jira.sakaiproject.org/browse/SAK-28990) , I could do the amend if required.